src/main/java/to/us/resume_builder/business/export_LaTeX/ResumeTemplate.java

```diff
@@ -74,10 +83,26 @@
                 categoryTemplates.put(c, new StringTemplate(""));
             }
 
+            LOGGER.severe("Could not read template files for template " + templateName);
+
             e.printStackTrace(); // TODO: display error message
         }
     }
 
+    private String readTemplate(String templateName, String fileName) throws IOException {
+        StringBuilder template = new StringBuilder();
+        URL path = Thread.currentThread().getContextClassLoader().getResource("templates/" + templateName + "/" + fileName);
+        try (InputStream in = path.openStream();
```

Comment:
It lets you have a try block with no catch? Mine's never let me do that.

Reply:
A try-with-resources clause only requires a `catch` block if an exception can be thrown within it.

Reply:
Well, fascinating. Carry on.